.travis.yml

```diff
@@ -5,16 +5,22 @@ before_script:
     - bash /tmp/bats/install.sh /tmp/local
     - export PATH=$PATH:/tmp/local/bin
     # install xspec
-    - git clone -b master https://github.com/xspec/xspec.git /tmp/xspec
-    # install saxon
+    - git clone -b $TRAVIS_BRANCH https://github.com/xspec/xspec.git /tmp/xspec
```

Comment:
You are right. Travis actually performed an automatic git clone right at the beginning of the job, I never noticed this - my bad!